lambdas/db-migration/src/migrations/20200911163145_create_providers_table.ts

```diff
@@ -0,0 +1,54 @@
+import * as Knex from 'knex';
+
+export const up = async (knex: Knex): Promise<void> =>
+  knex.schema.createTable('providers', (table) => {
+    table
+      .increments('cumulusId')
+      .primary();
+    table
+      .text('name')
+      .comment('Provider name')
+      .notNullable();
+    table
+      .enum('protocol', ['http', 'https', 'ftp', 'sftp', 's3'])
+      .comment('Protocol for the provider')
+      .notNullable();
+    table
+      .text('host')
+      .comment('Host name for the provider')
+      .notNullable();
+    table
+      .integer('port')
+      .comment('Port name for acessing the provider');
+    table
+      .text('username')
+      .comment('Username for acessing the provider');
+    table
+      .text('password')
+      .comment('password for acessing the provider');
+    table
+      .boolean('encrypted')
```

Comment:
It is so long as we have code like this using it:

https://github.com/nasa/cumulus/blob/master/packages/ingest/src/HttpProviderClient.js

I'm a little hesitant to change it because that goes beyond a simple "switch to reading to RDS instead of dymamo". It seems like something we could deprecate in the future, but I don't know if I want to add it to this work

Reply:
Until we update code to read from the DB, that provider code isn't reading from the DB. In more complex cases I agree that we should just lift and shift from Dynamo to RDS. This case seems trivial enough that creating that column and deleting it later without it ever being used seems like more work. I'll defer to @nemreid and @Jkovarik, though.

Reply:
39a7cfe091e64390e9196b171af342e5b08e8314
24936aa28b3c18cf2b0fbbe99b35d7fc7ce1adbf